### ISSUE TITLE
unable to write the Hamiltonian into a Device.TSHS file

### ISSUE THREAD

**Issue by tomato151** [zerothi/sisl#530]
**Describe the bug**
I create a Hamiltonian with the help of sisl and I fail to write it into a TSHS file. But it can be written in a nc file by H.write().
**Reproducable code**
```python
import sisl
graphene = sisl.geom.graphene(orthogonal=True)
H = sisl.Hamiltonian(graphene)
H.construct([[0.1, 1.43], [0., -2.7]])

tshs = sisl.io.tshsSileSiesta('Device.TSHS','w')
tshs.write(H)
```
It fails to create a ‘Device.TSHS’ file. 
Thank you in advance!
**Version details**
```python
3.8.5 (default, Sep  4 2020, 07:30:14)
[GCC 7.3.0]
0.10.0
```

**Comment by zerothi** [comment]
I believe this would work if you did:

```
tshs.write_hamiltonian(H)
```
Let me know if it works?

**Comment by zerothi** [comment]
Also, please update `sisl`, lots of progress has been made, and many bug-fixes since 0.10!

**Comment by tomato151** [comment]
It works! I will update sisl later. Thank you very much!
